vector/src/main/res/values/strings_riotX.xml

```diff
@@ -2,5 +2,6 @@
 <resources>
 
     <!-- Strings not defined in Riot -->
+    <string name="labs_allow_extended_logging">Help developers by allowing extended logging.</string>
```

Comment:
In the summary, it will be better to explicitly inform the user that message content or any other private data will not be logged, even if the is set to true. So I propose:
"Help developers by allowing extended logging, when you send a Rageshake."
"Even when set to true, the application does not log message contents or other private data."

And move this settings next to the rageshake setting?